clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -78,3 +78,24 @@ endmacro()
 # Always set SYCL_HAS_FP16 to true to assume SYCL device to support float16
 message("dpct.cmake: SYCL_HAS_FP16 is set true by default.")
 set(SYCL_HAS_FP16 TRUE)
+
+# Find oneAPI lib packages
+macro(DPCT_FIND_PACKAGE package)
```

Comment:
Pls add behavior test for this help function in https://github.com/oneapi-src/SYCLomatic-test